libs/components/toast/src/lib/modules/toast/toast.component.ts

```diff
@@ -45,13 +46,18 @@ export class SkyToastComponent implements OnInit, OnDestroy {
    * Specifies a `SkyToastType` type for the toast to determine the color and icon to display.
    */
   @Input()
-  public set toastType(value: SkyToastType) {
-    this._toastType = value;
+  public set toastType(value: SkyToastType | undefined) {
+    if (value === undefined) {
```

Comment:
Did strict mode prevent you from using `this.toastTypeOrDefault = value || SKY_TOAST_TYPE_DEFAULT`? If so, I'd prefer the ternary operator here:

```
this.toastTypeOrDefault = value === undefined ? SKY_TOAST_TYPE_DEFAULT : value;
```

Reply:
Nope. That was just me coding too fast.

Reply:
Fixed